src/assertions/employment_status_assertions.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+def estado_no_lista(statuses_url, status_id, header):
```

Comment:
Añadir prefijo que es de assertions, por ejemplo
```suggestion
def assert_estado_no_presente_en_lista(statuses_url, status_id, header):
```